cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Scanner.java

```diff
@@ -32,6 +32,7 @@
   /**
    * Closes the scanner and releases any resources.
    */
+  @Override
```

Comment:
`AutoCloseable.close()` has `throws Exception` and that would break compatibility if we don't override. Also as per javadoc in `AutoCloseable.close()`.

```
     * <p>While this interface method is declared to throw {@code
     * Exception}, implementers are <em>strongly</em> encouraged to
     * declare concrete implementations of the {@code close} method to
     * throw more specific exceptions, or to throw no exception at all
     * if the close operation cannot fail.
```